smart-contracts/test/Lock/erc721/approve.js

```diff
@@ -23,7 +23,7 @@ contract('Lock ERC721', (accounts) => {
     describe('when the token does not exist', () => {
       it('should fail', () => {
         return locks['FIRST']
-          .approve(accounts[2], accounts[1], {
+          .approve(accounts[2], 42, {
```

Comment:
Instead of hardcoding the tokenId in tests, maybe use `await locks['FIRST'].getTokenIdFor("ownerAddress")`

Reply:
lol I added this comment to the one fake tokenId example, but the point applies to other examples :)

Reply:
yes, got it. I started doing that, but never went back and modified tests I had already fixed. Will do that.